lib/project_types/script/layers/infrastructure/assemblyscript_project_creator.rb

```diff
@@ -90,7 +48,8 @@ def write_package_json
                 "assemblyscript": "^0.14.0"
               },
               "scripts": {
-                "test": "asp --config test/as-pect.config.js --summary --verbose"
+                "test": "asp --summary --verbose",
+                "build": "npx shopify-scripts-toolchain-as build --src src/script.ts --binary #{script_name}.wasm -- --lib node_modules --optimize --use Date="
```

Comment:
Nit: I don't think we need `npx` in here. It should just resolve to the correct binary without it.